core/src/main/scala/spire/math/ULong.scala

```diff
@@ -64,6 +64,7 @@ class ULong(val signed: Long) extends AnyVal {
     if (signed >= 0L) signed.toString
     else (ULong.LimitAsBigInt + signed).toString
 
+  // TODO: replace these with `===` and `=!=`? Don't we get `==` and `!=` for free with value classes?
```

Comment:
My incomplete understanding is that we get `==` and `!=` for free with a value class (like the above example). That's just an observation from my **minimal** usage of value classes.

If that's true, then, regardless of these `===` and `=!=` methods, why would `==` and `=!` need to be defined?

**Caveat** - I don't actually how know value classes' `==` and `!=` get implemented, such as in the above `LongWrapper `example.